### ISSUE TITLE
Latest babel-cli uses old / outdated deps which cause github to warn on security deps

### ISSUE THREAD

**Issue by shlomif** [babel/babel#10067]
## Bug Report

**Current Behavior**
babel-cli on npm is 2 years old and has this:

```
    "babel-cli": {
      "version": "6.26.0",
      "resolved": "https://registry.npmjs.org/babel-cli/-/babel-cli-6.26.0.tgz",
      "integrity": "sha1-UCq1SHTX24itALiHoGODzgPQAvE=",
      "requires": {
        "babel-core": "^6.26.0",
        "babel-polyfill": "^6.26.0",
        "babel-register": "^6.26.0",
        "babel-runtime": "^6.26.0",
        "chokidar": "^1.6.1",
        "commander": "^2.11.0",
        "convert-source-map": "^1.5.0",
        "fs-readdir-recursive": "^1.0.0",
        "glob": "^7.1.2",
        "lodash": "^4.17.4",
        "output-file-sync": "^1.1.2",
        "path-is-absolute": "^1.0.1",
        "slash": "^1.0.0",
        "source-map": "^0.5.6",
        "v8flags": "^2.1.1"
      }
    },

```

now chokidar ends up requiring braces below 2.3.1 which npm audit and github report as vulnerable.

See:

```
shlomif[fcs]:$trunk/fc-solve/site/wml$ npm audit
                                                                                
                       === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.11                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ lodash-cli                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ lodash-cli > lodash                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/782                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ braces                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=2.3.1                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ babel-cli                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ babel-cli > chokidar > anymatch > micromatch > braces        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/786                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 2 vulnerabilities (1 low, 1 moderate) in 7796 scanned packages
  2 vulnerabilities require manual review. See the full report for details.

```

on https://github.com/shlomif/fc-solve/tree/master/fc-solve/site/wml . Please update it on npm.

**Comment by nicolo-ribaudo** [comment]
This has been fixed in Babel 7.3.4. I don't think that we will release a new Babel 6 version (which is no longer maintained) just for this, since `babel-cli --watch` is only used during developmenet/deployment, and the only way an attacker could use this vulnerability to DoS you is by changing the paths you are passing to Babel as a cli argument.

**Comment by shlomif** [comment]
@nicolo-ribaudo : thanks! I have now upgraded to `@babel/cli` per this - https://github.com/shlomif/fc-solve/commit/a635aaec1d4a8fd3b0998325a23612beffc9320c .
